yellowbox/extras/redis/base.py

```diff
@@ -0,0 +1,46 @@
+from abc import abstractmethod
+from typing import IO, Optional, ClassVar
+
+from docker import DockerClient
+
+from yellowbox import RunMixin
+from yellowbox.containers import create_and_pull, get_ports, upload_file
+from yellowbox.retry import RetrySpec, Catchable
+from yellowbox.subclasses import SingleContainerService
+
+
+REDIS_DEFAULT_PORT = 6379
+DEFAULT_RDB_PATH = "/data/dump.rdb"
+
+
+class BaseRedisService(SingleContainerService, RunMixin):
+
+    health_exceptions: ClassVar[Catchable] = Exception
```

Comment:
docstring

Reply:
rethinking the default value, check if an empty tuple works, if not, leave it unassigned